Homebrew/Archetypes/Barbarian/Barbarian (Sacred Kin).xml

```diff
@@ -0,0 +1,144 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<compendium version="5">
+	<subclass baseclass="Barbarian" HB="HB">
+		<name>Barbarian (Sacred Kin)</name>
+		<spellAbility>Constitution</spellAbility>
```

Comment:
It may be useful to also mark all the sorcerer spells as sacred kin, jsut to be explicit

Reply:
Maybe, but there isn't a difference from the sorcerer spell list, so it seems like a duplication. Eldritch knight doesn't dupe the wizard spells, it just references you get them from that list

Reply:
It is explicitly listed in the spell, since it is a core book, I'm not sure if there is a real precedent since the core books are listed on the spells and Homebrew doesn't have this case yet. Don't really want to list every sorcerer spell, since it just adds another place it needs to be maintained.

I think that since this is a homebrew class that just gets, basically, all the spells from another class, the reference is fine for now, an enhancement request will be added to try to figure this out

Reply:
#139